src/sentry/api/endpoints/organization_stats.py

```diff
@@ -91,11 +93,19 @@ def get(self, request, organization):
         elif stat == 'generated':
             if group == 'project':
                 stat_model = tsdb.models.project
+                try:
+                    query_kwargs['environment_id'] = self._get_environment_id_from_request(
+                        request,
+                        organization.id,
+                    )
+                except Environment.DoesNotExist:
+                    raise ResourceDoesNotExist
 
         if stat_model is None:
             raise ValueError('Invalid group: %s, stat: %s' % (group, stat))
 
-        data = tsdb.get_range(model=stat_model, keys=keys, **self._parse_args(request))
+        data = tsdb.get_range(model=stat_model, keys=keys, **
+                              self._parse_args(request, **query_kwargs))
```

Comment:
This is the autoformatter's finest work yet

Reply:
Oh my god.

If you move the `**` down will it leave it there?

Reply:
Yep, that did it. Good thinking